Lib/typing.py

```diff
@@ -428,8 +428,15 @@ def __getitem__(self, parameters):
         return self._getitem(self, *parameters)
 
 
-@_SpecialForm
-def Any(self, parameters):
+class _AnyMeta(type):
```

Comment:
The metaclass is unfortunate because it restricts what classes can double-inherit from Any (due to metaclass conflicts). Seems unavoidable though.

Reply:
Not entirely, if we were willing to give up on instancecheck, I'd say we could just remove the metaclass entirely